File: articles/why-origin-stories-of-algorithms-need-care/c/origin_care_score.c
#include <stdio.h>

int main(void) {
    double scores[] = {0.96, 0.98, 0.96, 0.88, 0.98, 0.90, 0.90, 0.96, 0.98, 0.98};
    int n = sizeof(scores) / sizeof(scores[0]);
    double sum = 0.0;
    for (int i = 0; i < n; i++) sum += scores[i];
    printf("origin_care_score=%.6f\n", sum / n);
    return 0;
}

File: articles/why-origin-stories-of-algorithms-need-care/haskell/OriginCareScore.hs
module Main where

main :: IO ()
main = do
  let scores = [0.96, 0.98, 0.96, 0.88, 0.98, 0.90, 0.90, 0.96, 0.98, 0.98]
      score = sum scores / fromIntegral (length scores)
  putStrLn ("origin_care_score=" ++ show score)
